LeezeIt/LeezeIt/ViewController.swift

```diff
@@ -7,13 +7,131 @@
 
 import UIKit
 
-class ViewController: UIViewController {
 
+
+class ViewController: UIViewController {
+    
+    let parentStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.distribution = .fillEqually
+        stackView.axis = .vertical
+        stackView.spacing = 190
+        return stackView
+    }()
+    
+    let logoImageStackView:  UIStackView = {
+        let stackView = UIStackView()
+        return stackView
+        
+    }()
+    
+    let buttonStackView:  UIStackView = {
+        let stackView = UIStackView()
+        stackView.distribution = .fillEqually
+        stackView.axis = .vertical
+        stackView.spacing = 30
+        return stackView
+    }()
+    
+    var logoImage: UIImageView = {
+        let lztImage =  UIImageView(image: .init(named: "L"))
```

Comment:
please fix your naming convention, change the name of the constant as well as the name of the image

Reply:
resolved